fhnw-ph-schreibberatung.csl

```diff
@@ -0,0 +1,476 @@
+
+  <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="de-CH" class="in-text" demote-non-dropping-particle="sort-only">
+
+  <info>
+    <title>Schreibberatung Paedagogische Hochschule Fachhochschule Nordwestschweiz (Empfehlungen)</title>
+    <title-short>Schreibberatung PH FHNW</title-short>
+    <id>http://www.zotero.org/styles/fhnw-ph-schreibberatung</id>
+    <link href="http://www.zotero.org/styles/fhnw-ph-schreibberatung" rel="self"/>
+    <link href="http://www.schreiben.zentrumlesen.ch/stud_literaturverwaltungsprogramme_zotero.cfm" rel="documentation"/>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <author>
+      <name>Res Mezger</name>
+      <email>schreibberatung.ph@fhnw.ch</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <updated>2015-01-13T12:00:00+00:00</updated>
+    <summary>Empfehlungen der Schreibberatung PH FHNW (Paedagogische Hochschule, Fachhochschule Nordwestschweiz (CH))</summary>
+  </info>
+
+  <!-- "et al." anstatt "u.a." -->
+    <locale xml:lang="de">
+      <terms>
+        <term name="et-al">et al.</term>
+      </terms>
+    </locale>
+      
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="none">
+        <group delimiter=". ">
+          <choose>
+            <if variable="author">
+              <names variable="editor">
+                <label form="verb-short" text-case="capitalize-first" suffix=" " strip-periods="true"/>
+                <name and="text" delimiter=", "/>
+              </names>
+            </if>
+          </choose>
+          <choose>
+            <if variable="author editor" match="any">
+              <names variable="translator">
+                <label form="verb" text-case="capitalize-first" suffix=" " strip-periods="true"/>
+                <name and="text" delimiter=", "/>
+              </names>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+
+  <macro name="container-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <group delimiter=". ">
+          <choose>
+            <if variable="author">
+              <names variable="editor">
+                <name form="long" name-as-sort-order="all" and="text" delimiter="; "  delimiter-precedes-last="never"/>
+                <label form="short" prefix=" (" suffix=".)" strip-periods="true"/>
+              </names>
+            </if>
+          </choose>
+          <choose>
+            <if variable="author editor" match="any">
+              <names variable="translator">
+                <name form="long" name-as-sort-order="all" and="text" delimiter="; "   delimiter-precedes-last="never"/>
+                <label form="short" prefix=" (" suffix=".)" strip-periods="true"/>
```

Comment:
in addition to what Rintze says: could you take out all the combinations of a period in the suffix followed by strip-periods="true"? They're just clutter that we try to get rid of (i.e. what you have here is the same as
```<label form="short" prefix=" (" suffix=")"/>```